templates/contact/index.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+{% load addcss %}
+{% load render_bundle from webpack_loader %}
+
+{% block title %}
+Kontakt oss - Online
+{% endblock title %}
+
+{% block styles %}
+	{{ block.super }}
+	{% render_bundle 'contact' 'css' %}
+{% endblock %}
+
+{% block js %}
+	{{ block.super }}
+	{% render_bundle 'contact' 'js' %}
+{% endblock %}
+
+{% block content %}
+
+<section id="contact">
+	<div class="container">
+
+        <div class="row">
+            <div class="col-md-6">
+                <div class="page-header">
+                    <h1 id="resourcecenter-heading">Kontakt oss</h1>
+                </div>
+            </div>
+            <div class="col-md-6"></div>
+        </div>
+
+		<div class="row" id="introduction">
+            <div class="col-md-6">
+                <p>
+                    Ønsker du å komme med forslag til hovedstyret eller noen innspill til utviklingsteamet dotkom?
+                    Da kan du enkelt ta kontakt med oss via dette skjemaet!
+                </p>
+                <hr>
+            </div>
+            <div class="col-md-6"></div>
+		</div>
+
+		<div class="row">
+			<div class="col-md-6">
+				<form method="POST" id="contact_form" action="{% url 'submit' %}">
+					{% csrf_token %}
+                    <div id="choice_label">
+                    	{{ form.contact_receiver.label }}
+                    </div>
+                    {{ form.contact_receiver|addclass:"form-control" }}
+                    <hr>
+					{{ form.contact_name|addclass:"form-control" }}
+					{{ form.contact_email|addclass:"form-control" }}
```

Comment:
If we make the fields not required, which after thinking for a bit I think is a good idea, we don't want to do this. Keeping the fields there and changing the placeholder text to reflect the fields voluntary nature would make it clear to the user that you can leave an anonymous message even while logged in.